static/app/components/replays/replayController.tsx

```diff
@@ -156,10 +156,9 @@ const ReplayControls = ({
 
       <Button
         size="xs"
-        title={isFullscreen ? t('Exit full screen') : t('View in full screen')}
-        aria-label={isFullscreen ? t('Exit full screen') : t('View in full screen')}
+        title={isFullscreen ? t('Exit full screen') : t('Enter full screen')}
+        aria-label={isFullscreen ? t('Exit full screen') : t('Enter full screen')}
         icon={<IconResize size="sm" />}
-        priority={isFullscreen ? 'primary' : undefined}
```

Comment:
intentional?

Reply:
It was when I added the enter + exit full screen icons. Since there was separate icons didn't seem necessary to highlight the icon anymore. Added back for now